set_RMQ_EC2.py

```diff
@@ -0,0 +1,69 @@
+##Should run on local machine
+## And ssh into your AWS accounts then start rabbitmq-server
+
+import paramiko
+
+hostnames = [
+	'ec2-54-173-2-90.compute-1.amazonaws.com'		##Your instance's public name
+	
+]
+## since gonna be same for each host
+k = paramiko.RSAKey.from_private_key_file("/home/aniket/Downloads/rabbitmqkey.pem")
+
+##To set up RMQ-server up and running on each node
+for host in hostnames:
+	print "Setting RabbitMQ on host", host
+	## To establish ssh session with the reqd server
+	ssh = paramiko.SSHClient()
+	ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+	ssh.connect(hostname=host, username='ubuntu', pkey=k)
+	transport = ssh.get_transport()
+	session = transport.open_session()
+	session.set_combine_stderr(True)
+	session.get_pty()
+
+	## To execute the commands
+	commands =[
+		'sudo -i apt-get install rabbitmq-server -y',
+		'sudo rabbitmq-plugins enable rabbitmq_management',
+		'sudo -i rabbitmq-server -detached',
+		'sudo -i rabbitmqctl stop_app',
+		'sudo -i rabbitmqctl reset',
+		'sudo -i rabbitmqctl start_app',
+		'sudo -i rabbitmqctl cluster_status',
+	]
+
+	## To make a single command
+	single_command = ""
+	for i in range(len(commands)):
+		if i < len(commands)-1:
+			single_command=single_command + (commands[i]+"\n")
+		else:
+			single_command=single_command + (commands[i])
+
+	print single_command
+	session.exec_command(single_command)
+	stdin = session.makefile('wb', -1)
+	stdout = session.makefile('rb', -1)
+	shell_lines = stdout.readlines()
+	for s in shell_lines:
+		print s
+	ssh.close()
+
+'''
+Chain of Events
```

Comment:
Also: There are no multiline comments in python yet.
https://www.python.org/dev/peps/pep-0008/#block-comments

Also, fun: https://twitter.com/vinayakcoder/status/670178989500071937